presto-hive/src/test/sql/create-test.sql

```diff
@@ -237,7 +237,7 @@ ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTIC
 ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS FOR COLUMNS;
 
 
-CREATE TABLE presto_test_types_textfile (
+CREATE TABLE presto_test_types_orc (
```

Comment:
don't we need to modify tests to use that tables? Otherwise how do we verify results?

Reply:
Previously it would create & populate `presto_test_types_textfile` only and then create _orc, others based on that.

After the change the logic is the same. Just _orc is the first table created.